ui_framework/src/components/header/_header.scss

```diff
@@ -0,0 +1,158 @@
+// Header. Includes breadcrumbs of nav buttons.
+
+$kuiHeaderChildSize: $kuiSizeXXL + $kuiSizeL;
+
+@include component('kuiHeader') {
+  background: $kuiHeaderBackgroundColor;
+  border-bottom: $kuiBorderThin;
+
+  @include child('logo') {
```

Comment:
Talking offline. Agreed breadcrumbs should be broken up, but logo is just in use once here, so will remain part of the main one.